src/batch-client/index.ts

```diff
@@ -1,5 +1,6 @@
 import DataLoader from 'dataloader';
 import castArray from 'lodash/castArray';
+import differenceWith from 'lodash/differenceWith';
```

Comment:
update rollup.config.js to specify this as external method

Reply:
Removed